### PR TITLE
Provide more guide document for the potential contributors

### PR DESCRIPTION
**Pull request by LinuxSuRen**
fix https://github.com/kubesphere/ks-devops/issues/211

This PR can explain itself due to it is documentation contribution.

## Desired reviewers

* I changed some files which you might be interested in them
  * @JohnNiang @AbdelouahabMbarki @shihh
* I'm trying to fix an issue created by you
  * @FeynmanZhou

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,27 @@
...
 ## Get started
 
 ### Quick Start
 
-- Install KubeSphere via [kk](https://github.com/kubesphere/kubekey/) (or other ways).
+- Install KubeSphere via [kubekey](https://github.com/kubesphere/kubekey/) (or [other ways](docs/installation.md)).
 
   ```bash
-  kk create cluster --with-kubesphere nightly-$(date -d yesterday '+%Y%m%d')
+  kk create cluster
```
Comment (JohnNiang) [inline README.md:26]: Please note that this step only installs Kubernetes, not KubeSphere.

File: docs/cli.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+You can manipulate KubeSphere DevOps via CLI.
+
+## Create Pipeline
+
+```shell
+ks pip create --ws simple --template java --project default --skip-check -b good
```
Comment (JohnNiang) [inline docs/cli.md:6]: Before we use ks, I suggest introducing `ks` from `docs/installation.md#kubesphere-cli`.

File: docs/installation.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+### kubekey
+
+```shell
+ks install kk --components devops
+```
+
+### k3d
+
+k3d is a little helper to run Rancher Lab's [k3s](https://github.com/k3s-io/k3s) in Docker.
```
Comment (AbdelouahabMbarki) [inline docs/installation.md:36]: link to [k3d](https://github.com/rancher/k3d)  project

File: docs/installation.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+```shell
+ks install k3d --components devops
+```
+
+### kind
+
+kind is a tool for running local Kubernetes clusters using Docker container `nodes`.
```
Comment (AbdelouahabMbarki) [inline docs/installation.md:44]: ink to [kind](https://github.com/kubernetes-sigs/kind) project

**Review by AbdelouahabMbarki — COMMENTED** [review]
great ! i have left some comment

File: docs/installation.md [hunk 5]
```diff
@@ -33,15 +33,15 @@ ks install kk --components devops
 
 ### k3d
 
-k3d is a little helper to run Rancher Lab's [k3s](https://github.com/k3s-io/k3s) in Docker.
+[k3d]((https://github.com/rancher/k3d)) is a little helper to run Rancher Lab's [k3s](https://github.com/k3s-io/k3s) in Docker.
```
Comment (AbdelouahabMbarki) [inline docs/installation.md:36]: typo()

**Review by AbdelouahabMbarki — CHANGES_REQUESTED** [review]

**Review by AbdelouahabMbarki — APPROVED** [review]
/lgtm

**Comment by JohnNiang** [comment]
Please resolve the conflicting files first.

**Pull request merged**